Source/WebCore/Modules/mediastream/RTCPeerConnection.cpp

```diff
@@ -215,6 +215,42 @@ void RTCPeerConnection::createOffer(RTCOfferOptions&& options, Ref<DeferredPromi
         return;
     }
 
+    // https://www.w3.org/TR/webrtc/#legacy-configuration-extensions
```

Comment:
There are likely WPT tests that would benefit of this.
Can you verify this?

Reply:
I only found `imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-ontrack.https.html` but it's already passing with current main... 🤔